python/cugraph/cugraph/gnn/feature_storage/feat_storage.py

```diff
@@ -104,6 +133,31 @@ def _cast_feat_obj_to_backend(feat_obj, backend: str):
                 return _cast_to_torch_tensor(feat_obj.values)
             else:
                 return _cast_to_torch_tensor(feat_obj)
+        elif backend == "wholegraph":
+            wg_comm_obj = kwargs.get("wg_comm", wgth.get_local_node_communicator())
+            wg_type_str = kwargs.get("wg_type", "distributed")
+            wg_location_str = kwargs.get("wg_location", "cuda")
```

Comment:
I think we should  store these while initiating the FeatureStore object as I don't expect these to change b/w function calls.

Reply:
@VibhuJawa is it better now?